spec/rspec/matchers/expecteds_for_multiple_diffs_spec.rb

```diff
@@ -60,12 +60,14 @@ def create_matcher(stubs)
       end
 
       describe "#message_with_diff" do
-        it "returns just provided message if diff is empty" do
-          allow(FakeDiffer).to receive(:diff) { "" }
+        it "returns message and warning if diff is empty" do
```

Comment:
Suggest: `it "returns a message warning if the diff is empty" do`

Reply:
Good idea!